src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js

```diff
@@ -0,0 +1,349 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import uuidv4 from 'uuid/v4'
+import FancyAnimatedLogo from '../UniversalNavbar/FancyAnimatedLogo'
+import LogoNotAnimated from '../UniversalNavbar/assets/ethos-logo-black.js'
+import LogoWhite from '../UniversalNavbar/assets/ethos-logo-white.js'
+import {
+  AccountIcon,
+  SearchIcon,
+  AccordionToggleIcon,
+} from '../UniversalNavbar/assets/icons.js'
+import {
+  Body,
+  Button,
+  Layout,
+  Spacer,
+  TitleMedium,
+  TitleSmall,
+  Footnote,
+  COLORS,
+} from '../index'
+import TransformingBurgerButton from '../UniversalNavbar/TransformingBurgerButton/TransformingBurgerButton'
+import { LINKS } from './constants.js'
+import NavLink from '../UniversalNavbar/NavLink'
+import styles from './UniversalNavbarExpanded.module.scss'
+
+// TODO: convert from class to hook?
+class UniversalNavbarExpanded extends React.Component {
+  state = {
+    showMobileMenu: false,
+    activeAccordionItem: false,
+  }
+
+  toggleHamburger = () => {
+    this.setState({ ...this.state, showMobileMenu: !this.state.showMobileMenu })
+  }
+
+  handleHamburgerKeyPress = (event) => {
+    if (event.key === 'Enter') {
+      this.toggleHamburger(true)
```

Comment:
Looks like we don't need to pass in a boolean param

Reply:
yeah typo from temp code